src/Sarif.Viewer.VisualStudio/Converters/ImportanceToBackgroundConverter.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.Sarif.Viewer.Models;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Data;
+
+namespace Microsoft.Sarif.Viewer.Converters
+{
+    class ImportanceToBackgroundConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            var node = value as CallTreeNode;
+            return node.Location.Importance == AnnotatedCodeLocationImportance.Important ? Color.Yellow : Color.White;
```

Comment:
Instead of white for the default background color, should we use transparent instead?